openedx/features/job_board/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+from django_countries.fields import CountryField
+from model_utils.models import TimeStampedModel
+
+from .constants import JOB_COMPENSATION_CHOICES, JOB_HOURS_CHOICES, JOB_TYPE_CHOICES
+
+
+class Job(TimeStampedModel):
+    """
+    This model contains all the fields related to a job being
+    posted on the job board.
+    """
+    title = models.CharField(max_length=255, null=False, blank=False)
```

Comment:
Please remove `null=False, blank=False` as default value of null and blank is False.

Update other fields as well

https://docs.djangoproject.com/en/3.0/ref/models/fields/#django.db.models.Field.null
https://docs.djangoproject.com/en/3.0/ref/models/fields/#blank